components/automate-chef-io/content/docs/iam-actions.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "IAM Actions"
+description = "Reference Actions for Roles"
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "authorization"
+    weight = 30
++++
+
+Reference the chart on this page when creating a *Role* to know which action grants access to what page in the browser.
+
+*IAM Action* lists the associated action or actions required to access that page in the browser. 
+Use `*` in these actions to give broad permissions to perform all associated actions such as get, list, create, delete, etc.
+Specify the action to restrict user access to the specific action.
+
+
+|  Task           | Browser Tab     | IAM Action       | API endpoint  | URL       |
+| --------------- | --------------- | ---------------- | ------------- | --------- |
+| View Events | Dashboards | event:* | /event_feed | https://{{< example_fqdn "automate" >}}/dashboards/event-feed |
+| View Service Group Data | Applications | applications:*  | /applications/service-groups | https://{{< example_fqdn "automate" >}}/applications/service-groups |
+| View Client Runs | Infrastructure | infra:nodes:*   | /cfgmgmt/nodes | https://{{< example_fqdn "automate" >}}/infrastructure/client-runs |
+| View Chef Servers | Infrastructure | infra:servers:* | <endpoint> | https://{{< example_fqdn "automate" >}}/infrastructure/chef-servers |
+| List Reports | Compliance | compliance:reporting:*  | /compliance/reporting/reports | https://{{< example_fqdn "automate" >}}/compliance/reports/overview |
+| List Scan Jobs | Compliance | compliance:scannerJobs:* | /compliance/scanner/jobs | https://{{< example_fqdn "automate" >}}/compliance/scan-jobs/jobs |
+| Add or Edit Scan Jobs | Compliance | [compliance:scannerJobs:* , infra:nodes:* , infra:nodeManagers:* , compliance:profiles:* ] | /compliance/scanner/jobs | https://{{< example_fqdn "automate" >}}/compliance/scan-jobs/jobs |
```

Comment:
@susanev How does this change through the document look?

Reply:
the * gives list, get, create, delete permissions so i t think its a bit misleading, but i also understand yall are trying to highlight what is currently possible. so mixed feelings i guess. i would not try and block this from merging though. we could try this out and see if we get user feedback and adjust from there.

Reply:
That's why we prefer 'manage'; it implies the whole scope of actions.

Reply:
so they would all say manage?

Reply:
Everything from line 28 to line 40 would say "manage" rather than "add or edit" in this case.